### DISCUSSION TITLE
Should we unfork?

### DISCUSSION THREAD

**Discussion by jankapunkt** [node-oauth/node-oauth2-server#198] [General]
I found that this repo does not appear in any search on GitHub as it's still considered a fork, pointing to the upstream of oauthjs.
This also hides certain options, for example how it compares to general community standards.

If we want to be found in search and gain more traction we may unfork this repo. This simply removes the status of being a fork (which also removes oauthjs as main target when opening a new pull request) and keeps everything else as it is.

What do you think?

**Comment by martinssonj** [comment]
Please do that. That explains why it was sometimes hard to find this library on GitHub.

**Reply by jankapunkt** [reply]
@HappyZombies @jorenvandeweyer @Uzlopak any doubts or do you agree as well?

**Reply by jorenvandeweyer** [reply]
If this is this case we should definitely unfork the repo

**Reply by Uzlopak** [reply]
Yeah. Write a support ticket. Github support will remove the fork flag in few hours.

**Reply by jankapunkt** [reply]
done
